.gitattributes

```diff
@@ -1,2 +1,8 @@
+*.el text eof=LF
+*.js text eof=LF
+*.json text eof=LF
+*.ron text eof=LF
+*.rs text eof=LF
+*.toml text eof=LF
+*.ts text eof=LF
```

Comment:
I also wonder if it effectively undos the undoing of `line_ending = unix` in rustfmt.toml? Should we add it back?